epub33/rs/index.html

```diff
@@ -1866,6 +1866,9 @@ <h5>Rendering EPUB content document elements</h5>
 			<section id="sec-behaviors-interaction">
 				<h4>Interacting with the EPUB content document</h4>
 
+				<p class="note">Earlier versions of this specification included some information about embedded audio and 
+					video [epubmediaoverlays-32]. This feature has been deprecated.</p>
```

Comment:
We can't deprecate in a note, otherwise the deprecation doesn't have normative standing. I'd recreate 9.3.2 rather than completely deleting the section so it matches other deprecated features. I can't suggest a fix through the github interface, but put back something like:

```
<section id="sec-embedded-media">
	<h4>Embedded audio and video</h4>
	<p>Automatic playback of embedded audio and video is now <a data-cite="epub-33#deprecated">deprecated</a>.</p>
	<p>Refer to its definition in [[epubmediaoverlays-32]] for more information.</p>
</section>
```

Reply:
This change should also be integrated before merging.

Reply:
Done in https://github.com/w3c/epub-specs/pull/2402/commits/9d5bb342b48b788443f95e0cbda2721bd1795e52